src/librustc/hir/map/collector.rs

```diff
@@ -147,11 +149,25 @@ impl<'a, 'hir> NodeCollector<'a, 'hir> {
             (name1, dis1).cmp(&(name2, dis2))
         });
 
+        // We hash the names of all local source files so we don't have to
```

Comment:
Actually, I find this comment a bit unclear. What *exactly* are we hashing here? Are these the "post-remapping" names being hashed?

Reply:
Correct, yes. I'll update the comment.